packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -1,70 +1,124 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
-
+import uniqueid from 'lodash.uniqueid';
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
 import RadioField from '../../src/RadioField';
 import Radio from '../../src/Radio';
 
 window.matchMedia = () => ({ matches: true });
+jest.mock('lodash.uniqueid');
+
+let userAgentGetter;
+beforeEach(() => {
```

Comment:
We do not want to run these for every test
```suggestion
beforeAll(() => {
```

Reply:
i have made changes as per suggestions.